robottelo/ui/my_account.py

```diff
@@ -10,3 +11,48 @@ class MyAccount(Base):
     def navigate_to_entity(self):
         """Navigate to My Account page"""
         Navigator(self.browser).go_to_my_account()
+
+    def _update_helper(self, input_locator_name, new_value,
```

Comment:
It is not for org update, it is in **each** module. That is a basic idea of CRUD tests when you have simple create, update, delete, search and etc methods